.github/renovate.json

```diff
@@ -4,10 +4,6 @@
     "local>axonivy/renovate-config",
     "schedule:earlyMondays"
   ],
-  "baseBranches": [
-    "master",
-    "/^release\\/[0-9]{2}\\.[0-9]+/"
-  ],
```

Comment:
tried to exclude the baseBranches ... but it seems like this doesn't work at least it fails locally

```
❯ renovate-config-validator
 INFO: Validating .github/renovate.json
 WARN: Found errors in configuration
       "file": ".github/renovate.json",
       "warnings": [
         {
           "topic": "Configuration Error",
           "message": "packageRules[0]: You must configure baseBranches inorder to use them inside matchBaseBranches."
         },
         {
           "topic": "Configuration Error",
           "message": "packageRules[1]: You must configure baseBranches inorder to use them inside matchBaseBranches."
         }
       ]
```

Reply:
Ok; it also appears as warning on remote :cry: 

![config-warns](https://github.com/user-attachments/assets/21ac0fc4-d30e-42b1-8884-e0f8d6a0dd05)
![renovate-issues](https://github.com/user-attachments/assets/77feb787-e45b-4106-b088-af6895619024)

Reply:
however; release/10+12 is still listed ... and seems maintained by schedulded actions. maybe the config is in charge despite the warnings? :man_shrugging: